src/webvtt/data.rs

```diff
@@ -406,15 +406,25 @@ impl TimedEventInterface for WebVttCue {
 
 impl Display for WebVttCue {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(
+        writeln!(f)?;
```

Comment:
I think it'd be better to move the extra newline outside to the `Display` trait implementation for `WebVttSubtitle` instead, since the implementation for `WebVttCue` might be used to display individual cues outside the context of a subtitle file.

Reply:
Done ✅ I added the newline at `src/webvtt/data.rs` line 173, since that seemed like the most logical spot for it to me